layers/mldv2.go

```diff
@@ -0,0 +1,600 @@
+// Copyright 2018 Nine Internet Solutions AG. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math"
+	"net"
+	"time"
+
+	"github.com/google/gopacket"
+)
+
+const (
+	// S Flag bit is 1
+	sTrue uint8 = 0x8
```

Comment:
mind prefixing these with their type?  layers/ is a very large, single global namespace (mostly because I'm bad at planning ahead...)

```go
mldv2STrue = 0x8
mldv2SMask = 0x8
mldv2QRVMask = 0x7
```